engine/lite/runtime/src/main/scala/pl/touk/nussknacker/engine/lite/HttpConfig.scala

```diff
@@ -0,0 +1,3 @@
+package pl.touk.nussknacker.engine.lite
```

Comment:
why is it in this module? I'd expect it as private to App/embedded?

Reply:
Because we reuse it in embedded DM. BTW it looks like some common http binding settings. Maybe I should move it to some utils-internal and rename to HttpBindingConfig? or maybe I should duplicate it?

Reply:
TBH I'd prefer duplicating. But utils-internal is also okeeish

Reply:
For me duplication of this class is fine as well :+1: - duplicated